projects/storefrontlib/src/lib/ui/components/media/picture/picture.component.ts

```diff
@@ -27,6 +27,8 @@ export class PictureComponent implements OnChanges {
   mainImage;
   missingProductImgSrc = missingProductImgSrc;
 
```

Comment:
Just thinking here. There are many scenarios where this is used. Perhaps there could be an input so this feature can be switched off. The default could even be true but at least there is a way to do so.